### PR TITLE
docs: Make our README.md prettier (with a few added details)

### PR DESCRIPTION
**Pull request by matchai**
#### Description

- Rearrange the top of the README to have the links above the divider
- Add some line breaks in big chunks of text
- Add emoji to top-level headings 🙌 
- Build out our "Contributing" section to detail where we need help

[**📝 Click here to preview the new README.md**](https://github.com/starship/starship/blob/pretty-readme/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,8 +133,7 @@ This project follows the [all-contributors](https://github.com/all-contributors/
     <img width="100" src="media/icon.png" alt="Starship rocket icon">
 </p>
 
-## License
-
-[ISC Licensed](./LICENSE)
+## 📝 License
 
-Copyright © 2019-present, Matan Kushner
+Copyright © 2019-present, [Matan Kushner](https://github.com/matchai).<br>
```
Comment (johnletey) [inline README.md]: How about:

```suggestion
Copyright © 2019-present, Starship Contributors.<br>
```

Comment (matchai) [reply README.md]: The copyright holder for a work has to be either a person or a company.
Maybe one day, when Starship co. gets created. 😉

Comment (Zegnat) [reply README.md]: > […] has to be either a person or a company.

Or person**s**, plural, as is the case with Starship. As far as I could find, you do not have a CLA that assigns contributors’ copyright over to you. So, as an example, the commit aceda0052f7e946d9c43afad5ae440af3e6cd43e was not authored by you and someone else holds the copyrights. (Disclaimer: if the code in that commit is enough to qualify the author for copyright, that is for legal to figure out 🤷‍♀)

This is why you will see projects that do not work with copyright assignments state that the copyrights are held by the contributors, often with a link to [the contributors graph](https://github.com/starship/starship/graphs/contributors).

Comment (matchai) [reply README.md]: I was under the impression that contributing to a project with a pre-existing license would relinquish copyright to the person who had initially created the license, but after some research, it looks like that's not the case. I'll open up a PR to update this. 😄
Thank you for the correction @Zegnat!

**Pull request merged**